dialect/sql/sqlgraph/graph.go

```diff
@@ -157,17 +179,17 @@ func Neighbors(dialect string, s *Step) (q *sql.Selector) {
 			Join(match).
 			On(to.C(s.To.Column), match.C(pk1))
 	case r == M2O || (r == O2O && s.Edge.Inverse):
-		t1 := builder.Table(s.To.Table)
+		t1 := builder.Table(s.To.Table).Schema(s.To.Schema)
 		t2 := builder.Select(s.Edge.Columns[0]).
-			From(builder.Table(s.Edge.Table)).
+			From(builder.Table(s.Edge.Table).Schema(s.Edge.Schema)).
 			Where(sql.EQ(s.From.Column, s.From.V))
```

Comment:
```suggestion
			From(builder.Table(s.Edge.Table).
			Schema(s.Edge.Schema)).
			Where(sql.EQ(s.From.Column, s.From.V))
```

Add \n (for other places as well)

Reply:
@a8m for sure, just FYI gofmt will not end up formatting as you suggested, but more like this: 


```golang
    From(builder.Table(s.Edge.Table).
        Schema(s.Edge.Schema)).
    Where(sql.EQ(s.From.Column, s.From.V))
```

Since `.Schema`  is part of the From input, and not the From return. Let me know if that's okay and I'll change all of them :)

Reply:
Oh, right. I missed this. Thanks

Reply:
Cool cool, will keep them as is then.